File: Diving into JavaScript: For Beginners/step04/4.02/4.2.04.js
/*

Напишите программу, которая будет запрашивать у пользователя два целых числа, начальное значение A и конечное значение B. 
Затем программа должна вывести на экран все целые числа в диапазоне от B до A в обратном порядке.

Sample Input 1:

3
8

Sample Output 1:

8
7
6
5
4
3

Sample Input 2:

2
5

Sample Output 2:

5
4
3
2

*/

let a = Number(prompt());
let b = Number(prompt());

for (let i = b; i >= a; i--) {
  console.log(i);
}

File: Python. Part 1/step04/4.02/4.2.04.py
n = int(input())
if  n <= 24 and n >= 0:
    if 9 <= n <= 18:
        print("Магазин открыт.")
    elif n > 18 or n < 9:
        print("Магазин закрыт.")
else:
    print("Время указано неверно.")
